src/sharing/components/ShareAutosuggest.jsx

```diff
@@ -66,10 +66,10 @@ export default class ShareAutocomplete extends Component {
   onKeyPress = event => {
     // The user wants to add an unknown email
     if (
-      event.key === 'Enter' ||
-      (event.keyCode === 13 && this.state.inputValue !== '') ||
-      (event.key === 'Space' ||
-        (event.keyCode === 32 && /^.+@.+/.test(this.state.inputValue)))
+      ((event.key === 'Enter' || event.keyCode === 13) &&
+        this.state.inputValue !== '') ||
+      ((event.key === 'Space' || event.keyCode === 32) &&
+        /^.+@.+/.test(this.state.inputValue))
```

Comment:
It shall be fixed or squashed with the previous commit, if not in the really first commit that implies this file.